HISTORY.rst

```diff
@@ -1,6 +1,45 @@
 =======
 History
 =======
+3.41.0 (2012-08-04)
+-------------------
+
+Changes
+
+* CCI has a new Metadata ETL task that allows adding fields to existing layouts. Sections can be selected using an index (#2766)
+
+* We added a task to enable an Einstein prediction (thanks, @erikperkins!) (#2778)
```

Comment:
@jstvz @davidmreed was it intentional to add this task class without a task config?

Reply:
I think we should almost always add task configs.

Reply:
Yeah, I agree. I'm guessing it was an oversight but asked the question in case there's some special reason not to in this case.

Reply:
No. I will address this quickly.

Reply:
https://github.com/SFDO-Tooling/CumulusCI/pull/2786

Reply:
And tasks.rst auto-generates now, right?